index.rst

```diff
@@ -477,13 +510,15 @@ and the entrypoint:
     }
 
 Further Considerations
-^^^^^^^^^^^^^^^^^^^^^^
+----------------------
 
 Your service will eventually be set up to run as a Docker container
 under GCE.  This will require population of a ``Dockerfile`` and
 deployment description files in ``kubernetes``.  However, those files
 are not in scope for this document, and, in general, are expected to be
-added by the DM/SQuaRE team.
+added by the DM/SQuaRE team.  (If you use ``cookiecutter`` you will
```

Comment:
Maybe `use cookiecutter` could instead by a link to https://github.com/lsst-sqre/uservice-bootstrap